docker/docker-entrypoint.sh

```diff
@@ -38,7 +35,7 @@ fi
 
 if [[ "${OPERATING_MODE}" == "clustered" ]]; then
   echo "Starting keycloak-server on clustered mode..."
-  exec /opt/jboss/keycloak/bin/standalone.sh --server-config=standalone-ha.xml $@
+  exec /opt/jboss/keycloak/bin/standalone.sh --server-config=standalone-ha.xml -bmanagement=$INTERNAL_POD_IP -bprivate=$INTERNAL_POD_IP -u 230.0.0.4 $@
```

Comment:
Do we actually need to set -u 230.0.0.4? It looks like 230.0.0.4 is used in standalone(-ha).xml by default anyway?